### PR TITLE
chore(extensions): add extension recommendations

### PR DESCRIPTION
**Pull request by JFrankfurt**

### CODE REVIEW

File: .vscode/extensions.json [hunk 1]
```diff
@@ -0,0 +1,10 @@
+{
+  "recommendations": [
+    "dbaeumer.vscode-eslint",
+    "eamodio.gitlens",
+    "esbenp.prettier-vscode",
```
Comment (zzmp) [inline .vscode/extensions.json:5]: I thought we ran prettier through eslint. Does it need its own extension? What does this add?

File: .vscode/extensions.json [hunk 2]
```diff
@@ -0,0 +1,10 @@
...
+    "jpoissonnier.vscode-styled-components",
+    "rvest.vs-code-prettier-eslint"
```
Comment (zzmp) [inline .vscode/extensions.json:7]: How come prettier isn't already enabled through vscode-eslint? What does this add?

File: .vscode/extensions.json [hunk 3]
```diff
@@ -0,0 +1,10 @@
+{
+  "recommendations": [
+    "dbaeumer.vscode-eslint",
+    "eamodio.gitlens",
```
Comment (zzmp) [inline .vscode/extensions.json:4]: What does this add to _your_ workflow above the native vs code git integrations?

**Comment by JFrankfurt** [comment]
@grabbou we can drop this

I made the PR before we had any extension recs, and I'm fine with just using what we have now

**Pull request closed without merging**